src/libraries/System.IO.Packaging/src/System/IO/Packaging/ContentType.cs

```diff
@@ -86,28 +86,20 @@ internal ContentType(string contentType!!)
                 ValidateCarriageReturns(contentType);
 
                 //Begin Parsing
-                int semiColonIndex = contentType.IndexOf(SemicolonSeparator);
```

Comment:
nit: when we had the `SemicolonSeparator` const it was easier to find the place where the string is getting concatenated

Reply:
And harder to read and to know what this actually was, its type, etc.